configs/katello_33.yaml

```diff
@@ -0,0 +1,178 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.2.3:
+    :redmine_release_id: 214
+:releases:
+  :3.3:
+    :redmine_release_id: 188
+:code_name: Baltic Porter
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.3
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
+    :branch: KATELLO-3.3
+    :repo: https://github.com/Katello/katello-selinux.git
+  :katello-installer:
+    :branch: KATELLO-3.3
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :hammer-cli-katello:
+    :branch: 0.1.0
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello.org:
+    :branch: KATELLO-3.3
+    :repo: https://github.com/Katello/katello.org.git
+  :katello-packaging:
+    :branch: KATELLO-3.3
+    :repo: https://github.com/Katello/katello-packaging.git
+    :version_branch: true
+  :puppet-katello:
+    :branch: 5.1.1
+    :repo: https://github.com/Katello/puppet-katello.git
+  :puppet-capsule:
+    :branch: 3.1.2
+    :repo: https://github.com/Katello/puppet-capsule.git
+  :puppet-pulp:
+    :branch: 3.4.0
+    :repo: https://github.com/Katello/puppet-pulp.git
+  :katello-agent:
+    :branch: 2.6.0
```

Comment:
This is up to 2.7.0. Looking at the repo, we should release a new one with the tracer support.